cpp/example/super_capacitor.info

```diff
@@ -0,0 +1,99 @@
+type SuperCapacitor
+dim  2
+verbosity 1
```

Comment:
is that the only difference with the input file we usually have? if the answer is yes, don't add a new file, parse the old one and put the extra data in the ptree.